src/os/ui/menu/areaexportmenu.js

```diff
@@ -0,0 +1,99 @@
+goog.provide('os.ui.menu.areaExport');
```

Comment:
One other structural change I would make would be to move all of this menu code into the `areas.js` file. The global, singleton menu design (i.e. instantiated in `MainCtrl` is only necessary for menus that are associated with UIs that are always in the view. Stuff like `MapMenu` and `SpatialMenu` for map interactions, and the menus on the top nav bar fall into this category. This one should be created and destroyed with the `areas.js` class.